### ISSUE TITLE
wasm64  was an error running eglGetDisplay

### ISSUE THREAD

**Issue by wph03634** [emscripten-core/emscripten#19324]
**Version of emscripten/emsdk:**
 3.1.35

**Failing command line in full:**
emcc test.cpp --bind -s MAIN_MODULE=1  -s MEMORY64=2 -s WASM=1 -o test.html

**Code- test.cpp:**
```
int main()
{ 
	EGLDisplay display  = eglGetDisplay(EGL_DEFAULT_DISPLAY);
        return 0;
}
```
**JS Error:**
Uncaught TypeError: Cannot convert 62000 to a BigInt

I have compiled wasm64, but JS is wrong, there was an error running eglGetDisplay, and wasm32 does not have this error./(ㄒoㄒ)/~~

**Comment by sbc100** [comment]
This has been fixed in #19259, but we still haven't done a release that includes that change.   You can try it out with `./emsdk install tot`, and we can do a release of 3.1.38 soon that will include it.

**Comment by sbc100** [comment]
Closing for now, since I verified that fix locally.
